examples/zsec

```diff
@@ -326,36 +326,59 @@ done
     cc_instance_size=small
     fi
 
-ccvm_instance_type_default=m5.large
+if [[ $cc_instance_size == "small" ]]; then
+ccvm_instance_type_default="m5.large"
 while true; do
-    read -r -p "Enter desired AWS EC2 instance type for CC. Recommended types: Small CC (m5.large or c5.large); Medium (m5.2xlarge or c5.2xlarge); Large (m5.4xlarge or c5.4xlarge) [Default=$ccvm_instance_type_default]: " ccvm_instance_type_input
-ccvm_instance_type=${ccvm_instance_type_input:-$ccvm_instance_type_default}    
+    read -r -p "Enter desired AWS EC2 instance type for $cc_instance_size CC. Recommended types: (m5.large or c5.large) [Default=$ccvm_instance_type_default]: " ccvm_instance_type_input
+ccvm_instance_type=${ccvm_instance_type_input:-$ccvm_instance_type_default}
     case $ccvm_instance_type in
       t3.medium|m5.large|c5.large|c5a.large|m5.2xlarge|c5.2xlarge|m5.4xlarge|c5.4xlarge)
```

Comment:
we only support 1 flavor of AMD?  c5a.large?

Reply:
Yeah that's probably a QA question